api/routes/A1/Users.php

```diff
@@ -39,6 +41,7 @@ public function get($id)
     {
         $user = (array)$this->usersGateway->getEntries(['id' => $id, 'status' => null]);
 
+        $this->setTags(['table_users', 'entity_user_'.$id]);
```

Comment:
We were expecting the new layer to cache the output, so we don't hit the db, but acting everything else as normal, so the user can execute hooks. that's one cache, then we would have a second one that's what you are using and that's the HTTP cache, unless this has not been clear to me, if it's please let us know.